example-playbooks/kernel_version_enforcer/ubuntu/main.yml

```diff
@@ -0,0 +1,118 @@
+---
```

Comment:
I don't think this functionality should be inside a playbook. We will want to re-use this functionality when setting up a ScyllaDB node.